src/responder/kcm/kcmsrv_ccache_mem.c

```diff
@@ -414,11 +415,20 @@ static struct tevent_req *ccdb_mem_getbyuuid_send(TALLOC_CTX *mem_ctx,
     ccwrap = memdb_get_by_uuid(memdb, client, uuid);
     if (ccwrap != NULL) {
         state->cc = kcm_ccache_dup(state, ccwrap->cc);
+        if (state->cc == NULL) {
+            ret = ENOMEM;
+            goto fail;
```

Comment:
I'd use goto end/finish and avoid duplicating the `tevent_req_post(req, ev);` and `return req;` parts ...